contracts/TestInterfaces/IRewardDistributorTest.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity ^0.5.16;
 
-interface IRewardDistributor {
+interface IRewardDistributorTest {
```

Comment:
Why is `IRewardDistributorTest` needed? There should already be an `IRewardDistributor` in contracts/TestInterfaces/IRewardDistributor.sol with the exact same interface.

Reply:
So this is what was previously `IRewardDistributor`, same one your are referring to that is in contracts/TestInterfaces. I renamed it since it is only used for testing the v1 Reward Distributor. This is different than the one I made for JoeLensV2 though, since it defines `claimReward()` and `accrueReward()`, while the interface I created (now called IRewardDistributor), defines `rewardSupplySpeeds()` and `rewardBorrowSpeeds()`. 

I think we would want to keep both of them since RewardDistributorV2 function naming is different than in v1, i.e. `claimReward` in v1 and `_claimReward` in V2. Let me know if that makes sense

Reply:
I would remove IRewardDistributorTest.sol and add the functions to your IRewardDistributor.sol then.